test-data-setup/hmpps-integration-api/create-sample-crns-for-testing.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test } from '@playwright/test'
+import { login as loginDelius } from '../../steps/delius/login'
+import { createOffender } from '../../steps/delius/offender/create-offender'
+import { createAndBookPrisoner } from '../../steps/api/dps/prison-api'
+import { deliusPerson } from '../../steps/delius/utils/person'
+
+test('create a crn for Delius and DPS but not OASyS', async ({ page }) => {
+    await loginDelius(page)
+    const person = deliusPerson({ sex: 'Male', dob: null, lastName: 'Smith', firstName: 'Bob' })
```

Comment:
Do you need a fixed name of Bob Smith? If you omit the fields it'll create a new random realistic name each time.

Reply:
We put the fixed name in because when consumers of our API initially look for a person, they will search on first and last names. Specifying this just made it easier to find. I can omit this provided the first and last name are output in the report?

Reply:
I've omitted it for now. Will see how it runs